### PR TITLE
1.5- Week 1 Step 5 Added step to deploy function for testing

### PR DESCRIPTION
**Pull request by creaturenex**
## Changes made
* Added checkbox to test locally and then when confirmed working to deploy the function.
* Added reminder that testing locally uses F5 and testing remotely needs the function to be deployed.
* These instructions are meant to show the proper order of operation for students to test their code.

Closes [#280](https://github.com/bitprj/Intro-To-Serverless/issues/280)

### CODE REVIEW

File: .bit/responses/1.3-Week 1 Step 3.md
Comment (blueboxes) [inline .bit/responses/1.3-Week 1 Step 3.md:18]: Comment the picture?

File: .bit/responses/1.4-Week 1 Step 4.md
Comment (blueboxes) [inline .bit/responses/1.4-Week 1 Step 4.md:24]: I know this is same as before but I think it should say 'do not merge it yet' to show it will be merged at some point.

**Review by blueboxes — CHANGES_REQUESTED** [review]

File: .bit/responses/1.4-Week 1 Step 4.md
Comment (creaturenex) [reply .bit/responses/1.4-Week 1 Step 4.md:24]: No problem I'll update that. My question though is at what point do we no longer say do not merge? We as the user should not be merging until we see the gif TIME TO MERGE given by the bot.

File: .bit/responses/1.3-Week 1 Step 3.md
Comment (creaturenex) [reply .bit/responses/1.3-Week 1 Step 3.md:18]: I think what is meant. _Please respond by posting a comment. Within the comment attach the image you received from the API in this issue/pull request to move on_ 

Let me know if that si clearer to you and ill update that language

File: .bit/responses/1.4-Week 1 Step 4.md
Comment (creaturenex) [reply .bit/responses/1.4-Week 1 Step 4.md:24]: Where exactly do you want me to add that? & Should we use the same language everywhere?
W1S3 - does NOT say "do not merge"
W1S4 - already has the statement do not merge it
W1S5 - has the last line as ❗ Do not merge the branch!!

File: .bit/responses/1.3-Week 1 Step 3.md
Comment (emsesc) [reply .bit/responses/1.3-Week 1 Step 3.md:18]: @creaturenex Maybe be more concise by saying: `On this issue or pull request, please post a comment with the cat picture received from the API.`

File: .bit/responses/1.4-Week 1 Step 4.md
Comment (emsesc) [reply .bit/responses/1.4-Week 1 Step 4.md:24]: @creaturenex Good point, we should be more consistent about that. Also, we are adding a feature to the bot that doesn't let them merge until the bot approves it. Here's what we can do: put the key words "do not merge this branch until the bot tells you to do so!" every time we ask students to make a pull request.

File: .bit/responses/1.3-Week 1 Step 3.md
Comment (blueboxes) [reply .bit/responses/1.3-Week 1 Step 3.md:18]: Sorry for the late reply, count of context `Comment the picture` seemed odd,  I agree with @emsesc  I think her suggestion makes it clearer. I initially thought it meant write "I have a picture of a cat" 🤣

File: .bit/responses/1.4-Week 1 Step 4.md
Comment (blueboxes) [reply .bit/responses/1.4-Week 1 Step 4.md:24]: Happy to approve once the minor changes are made

Comment (creaturenex) [reply .bit/responses/1.4-Week 1 Step 4.md:24]: I'ill get to this later today thank you for responding

**Pull request closed without merging**
